token-address/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+
+# TODO: Delete me
+[package]
```

Comment:
I took this package from [here](https://github.com/p2p-org/token-service/tree/main/token-address)